### PR TITLE
Update Index.html

### PR DESCRIPTION
**Pull request by RahulYadav4Ads**

### CODE REVIEW

**Review by Rahul4ios — COMMENTED** [review]
Question from Me

**Comment by RahulYadav4Ads** [comment]
> Question from Me

Answer from Other

File: Index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <body style="text-align: center;">
 	<h1>
-		Geeks for Geeks
+		Geeks for Geeks!
```
Comment (Rahul4ios) [inline Index.html:6]: Why !

Comment (Rahul4ios) [inline Index.html:6]: May I know good reason behind ! please

**Review by Rahul4ios — CHANGES_REQUESTED** [review]

**Pull request merged**